lib/pf/UnifiedApi/Controller/Reports.pm

```diff
@@ -47,32 +47,92 @@ sub osclass_active {
 
 sub inactive_all {
```

Comment:
I think everything is this file could be removed (like pfcmd/reports.pm) since they are now all in the report.conf.defaults